sdk/face/ai-vision-face-rest/samples-dev/aadAuth.ts

```diff
@@ -5,6 +5,9 @@ import createFaceClient, {
     isUnexpected,
 } from '@azure-rest/ai-vision-face';
 
+import * as dotenv from "dotenv";
```

Comment:
Can we just use `import "dotenv/config";` and not need the second line?

Reply:
I referenced other SDK samples that use import `import * as dotenv from "dotenv"; dotenv.config();`. The code runs successfully with just `import 'dotenv/config';`. Should I update it to use the second approach?

Reply:
@mpodwysocki Any ideas?

Reply:
@mpodwysocki We have updated by your comments, please re-review this PR. Thanks a lot.